utils/catatonit/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=catatonit
+PKG_VERSION:=0.1.7
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/openSUSE/catatonit.git
+PKG_SOURCE_DATE:=2022-03-07
+PKG_SOURCE_VERSION:=d8d72fea155c144ed3bf298a35a1aba5625a5656
+PKG_MIRROR_HASH:=skip
+
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later
```

Comment:
It's actually GPL-3.0-or-later.
See: https://github.com/openSUSE/catatonit/blob/d8d72fea155c144ed3bf298a35a1aba5625a5656/COPYING

Reply:
Hey, that's right. Sorry, my bad. Probably was a typo in the morning (had 5 minutes time before had to start work so it was a bit sloppy work) - fixing it now..

Reply:
done..